imjoy/imjoyUtils3.py

```diff
@@ -58,12 +58,11 @@
                                 result = await result
                             resolve(result)
                         except Exception as e:
+                            traceback_error = traceback.format_exc()
                             logger.error(
-                                "error in method %s: %s",
-                                d["name"],
-                                traceback.format_exc(),
+                                "error in method %s: %s", d["name"], traceback_error
                             )
-                            reject(e)
+                            reject(Exception(formatTraceback(traceback_error)))
```

Comment:
The error string will be formatted and convert into an Exception, the reject function will encode it and send it as an Error object.